cmd/storagenode/main.go

```diff
@@ -43,6 +43,11 @@ var (
 		RunE:        cmdSetup,
 		Annotations: map[string]string{"type": "setup"},
 	}
+	configCmd = &cobra.Command{
+		Use:   "config",
+		Short: "Edit config files",
+		RunE:  cmdConfig,
```

Comment:
You need to add `Annotations: map[string]string{"type": "setup"},` here to make `setup` working inside this command.